examples/parallel_for/seismic/main.cpp

```diff
@@ -106,8 +120,13 @@ int main(int argc, char *argv[]) {
             else {
                 oneapi::tbb::global_control c(oneapi::tbb::global_control::max_allowed_parallelism,
                                               p);
-                for (int i = 0; i < numberOfFrames; ++i) {
-                    u.ParallelUpdateUniverse();
+                mu.clear();
```

Comment:
We can move instantiation of the `utility::measurements mu` here and have instead `double rel_error` created above. That will allow us dropping `utility::measurements::clear()` method.
```suggestion
                utility::measurements mu;
```
Then, just after the measurement loop, assign the `mu.computeRelError()` to that global `rel_error`.